### ISSUE TITLE
same isssue unable to connect

### ISSUE THREAD

**Issue by lynx-gif** [NimaNzrii/comfyui-photoshop#151]
![Screenshot 2024-07-16 191550](https://github.com/user-attachments/assets/b4caa5cd-9c6b-462a-850f-4f9f042a05b8)

**Comment by NimaNzrii** [comment]
can you send me your log file please?

**Comment by aazj2024** [comment]
The workflow still cannot be loaded
![QQ截图20240716225905](https://github.com/user-attachments/assets/7325a2af-49ef-4607-9cb4-e23e16d7069e)

**Comment by aazj2024** [comment]
The workflow still cannot be loaded
![1111](https://github.com/user-attachments/assets/ae849ec7-e4c7-4e77-bb18-f5cf34217f77)

**Comment by NimaNzrii** [comment]
Is your plugin update too?

**Comment by lynx-gif** [comment]
please make a tutorial and help us out and info regrading new updates too

**Comment by aazj2024** [comment]
Yes, the 1.6 version has been updated, the background has also been updated, and it is possible to connect, but it still cannot load the workflow




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "NimaNzrii/comfyui-photoshop"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年7月16日(星期二) 晚上11:30
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [NimaNzrii/comfyui-photoshop] same isssue unable to connect (Issue #151)





 
Is your plugin update too?

**Comment by aazj2024** [comment]
> Is your plugin update too?

Yes, the 1.6 version has been updated, the background has also been updated, and it is possible to connect, but it still cannot load the workflow

**Comment by cdmusic2019** [comment]
I've tried both methods of installation, unzipping 'CCX' and putting it in 'Plug-ins', which would not connect, but installing it with 'ZXP UXP Installer.' works fine and runs fine.

**Comment by smalltotal** [comment]
![微信截图_20240719094101](https://github.com/user-attachments/assets/3dc3552a-a27d-43cc-b433-12900e4c3746)
I encountered the same problem，please

**Comment by smalltotal** [comment]
If you roll back the plugin to version 6.29, you can connect to comfyui, but you still cannot get the image information.

**Comment by NimaNzrii** [comment]
@smalltotal @cdmusic2019 @aazj2024 @lynx-gif 

Guys sorry for this bug , there is no info about why it's happening

Is there anyway for aomeone to get more in touch with me in discord , so i can find the prob and fix it for you?

Discord id : NimaNzrii

**Comment by KINGLIFER** [comment]
I think your workflow is conflicting with other nodes. I found one on my setup: **dzNodes: LayerStyle**
Not sure if that is a culprit but I saw some references to it when I try to connect the plugin.

Perhaps we can roll back to the older one you had that was working? Or it could be ComfyUi updates that happened recently.

**Comment by NimaNzrii** [comment]
> I think your workflow is conflicting with other nodes. I found one on my setup: **dzNodes: LayerStyle** Not sure if that is a culprit but I saw some references to it when I try to connect the plugin.
> 
> Perhaps we can roll back to the older one you had that was working? Or it could be ComfyUi updates that happened recently.

May I know what made you notice this conflict between nodes?

I do not recommend rollback at all

**Comment by NimaNzrii** [comment]
@smalltotal @KINGLIFER @cdmusic2019 @aazj2024 try right clicking on the node and press install 
worked for somone

**Comment by locsmax** [comment]
> @smalltotal @KINGLIFER @cdmusic2019 @aazj2024Intente hacer clic derecho en el nodo y presione instalar. A alguien le funcionó.

Thanks! That solution worked for me too.
